### ISSUE TITLE
Events other than scroll or zoom do not work in bundle mode. 

### ISSUE THREAD

**Issue by redelman431** [neuronetio/gantt-elastic#7]
**Describe the bug**
When using event listeners other than the scroll or zoom in bundle mode, the event listeners such as click or mouseover do not fire when the specified components are either clicked or hovered over. 

**To Reproduce**
1. Modify the ready function in the examples/index.html file in your project. Do not use any Vue.js in that file. Only use VanillaJS or jQuery. 
2. Append these event listeners as shown here after line 266 of the examples/index.html file.

````
`GanttElastic.component.components['gantt-header'] = Header;
    const app = GanttElastic.mount({
      el: '#app', // <- your container id
      tasks: tasks,
      options: options,
      ready(ganttInstance) {
        console.log('ready!', ganttInstance);
        ganttInstance.$on('times-timeZoom-change', (ev) => {
          console.log('time zoom changed!', ev)
        });
        ganttInstance.$on('chart-task-click',function({event,data}) {
            console.log("click");
        });
        ganttInstance.$on('chart-milestone-click',function({event,data}) {
            console.log("click");
        });
        ganttInstance.$on('chart-project-click',function({event,data}) {
            console.log("click");
        });
        ganttInstance.$on('chart-task-mouseover',function({event,data}) {
            console.log("mouseover");
        });
        ganttInstance.$on('chart-milestone-mouseover',function({event,data}) {
            console.log("mouseover");
        });
        ganttInstance.$on('chart-project-mouseover',function({event,data}) {
            console.log("mouseover");
        });
   }
   });
  </script>
</body>

</html>
 
````
**Expected behavior**
Clicking or hovering on the chart project, tasks, and milestones should output the corresponding console.logs in each event. For example a hover should output "mouseover" in the console and a click should output "click". 

**Desktop (please complete the following information):**
 - OS: Linux and Mac. 
 - Browser: Chrome  66.0.3359.181 64-bit and Firefox 60.0 64-bit.

**Comment by neuronetio** [comment]
Yeah, I did not test bundle, I will try to look at this tomorrow
Thanks for report!

**Comment by redelman431** [comment]
No problem.

**Comment by neuronetio** [comment]
Fixed in 0.6.2
